src/data/content/risorse-per-progettare/realizzare/migrazione-sito.yaml

```diff
@@ -0,0 +1,273 @@
+# INSTRUCTIONS
+# folder structure:
+# levelX.yaml < contents and settings
+# levelX/ < folder with childrens
+# important: each level has its dedicated template, from level1 to level4
+#
+# comments:
+# #C to locate all editable content
+# #M for links/navigation
+# #I for images/media
+# #AUTO for parts that will soon be automated, preferably
+#
+# comment with # to hide something
+# delete with caution
+
+# ————————————————————————————————————————————
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  template:
+    name: level3
+  redirect_from:
+    - /kit/migrazione-sito
+    - /migrazione-sito
+    - /kit/migrazione-sito
+    - /kit/migrazione-sito
+    - /kit/migrazione-sito
+
+# ————————————————————————————————————————————
+# SEO, OPENGRAPH
+# ————————————————————————————————————————————
+seo:
+  name: Migrazione sito - Designers Italia #C REMIND "- Designers Italia" #AUTO
+  description: "Accompagna la transizione da un vecchio a un nuovo sito predisponendo URL e breadcrumbs chiari e reindirizzamenti corretti." #C
+  image: https://designers.italia.it/images/risorse-per-progettare/icons/kit_migrazione.png #I
```

Comment:
Questa immagine non esiste al momento. 
Qui deve andare il link all'immagine social del kit da fare e caricare in questa PR. Vedi commento sopra.